amundsen_application/static/js/components/ProfilePage/index.tsx

```diff
@@ -46,25 +45,42 @@ interface DispatchFromProps {
   getBookmarksForUser: (userId: string) => GetBookmarksForUserRequest;
 }
 
-export type ProfilePageProps = StateFromProps & DispatchFromProps;
+interface RouteProps {
+  userId: string;
+}
+
+
+export type ProfilePageProps = StateFromProps & DispatchFromProps & RouteComponentProps<RouteProps>;
 
 export class ProfilePage extends React.Component<ProfilePageProps> {
   private userId: string;
```

Comment:
Is there any reason why we use `private userId: string;` rather than using the `props` or `state`?